compatibility-with-other-plugins/compatible/graves.md

```diff
@@ -1,17 +1,21 @@
-# Graves
+# Graves/GravesX
 
-## [Download here](https://www.spigotmc.org/resources/graves.74208/)
+## [Graves Legacy Download here](https://www.spigotmc.org/resources/graves.74208/)
+## [GravesX Download here](https://www.spigotmc.org/resources/graves.74208/)
 
 ## How to add compatibility?
 
-Enable the compatibility
+> [!NOTE]
+> Graves configuration is located in ``/plugins/Graves`` while GravesX configuration is located in ``/plugins/GravesX`` while both plugins retain the same file structure besides plugin name differences.
```

Comment:
```suggestion
> Graves configuration is located in `/plugins/Graves` while GravesX configuration is located in `/plugins/GravesX` while both plugins retain the same file structure besides plugin name differences.
```
Inline code only needs a single back tick for start and end unless back ticks are inside the code.